ui/app/AppLayouts/Onboarding2/KeycardCreateProfileFlow.qml

```diff
@@ -123,9 +124,17 @@ SQUtils.QObject {
     Component {
         id: backupSeedRevealPage
         BackupSeedphraseReveal {
-            seedWords: root.seedWords
+            Component.onCompleted: {
+                try {
+                    const seedwords = root.getSeedWords()
+                    d.seedWords = JSON.parse(seedwords)
+                    root.seedphraseSubmitted(d.seedWords)
```

Comment:
This should be emitted only after the user has confirmed and moved to the next page

Reply:
1. `GenerateMnemonic` (which is called by `getSeedWords` is a pretty fast procedure. But I agree that even in this case it should be done asyncronously.
I think it's fine to merge as is and improve after, wdyt?

2. Not sure what you mean is wrong here? 🤔  
I don't get why we need to `JSON.parse`, I'll double check and remove it.

cc @micieslak